lms/static/lms/js/require-config.js

```diff
@@ -224,6 +225,9 @@
             // global namespace instead of being registered in require.
             'draggabilly': {
                 exports: 'Draggabilly'
+            },
+            'hls': {
+                exports: 'Hls'
```

Comment:
`Hls` is name which exposed on `window` like `window. Hls`. [Details](http://stackoverflow.com/a/29188269)